lib/datasource/npm/get.ts

```diff
@@ -75,15 +74,19 @@ export async function getDependency(
   if (cachedResult) {
     return cachedResult;
   }
-  const authInfo = registryAuthToken(regUrl, { npmrc });
   const headers: OutgoingHttpHeaders = {};
+  let authInfo = registryAuthToken(regUrl, { npmrc, recursive: true });
+
+  if (
+    !authInfo &&
+    npmrc &&
+    npmrc._authToken &&
+    regUrl.replace(/\/?$/, '/') === npmrc.registry?.replace(/\/?$/, '/')
+  ) {
+    authInfo = { type: 'Bearer', token: npmrc._authToken };
```

Comment:
FYI I *think* I recall that the registry-auth-token library added this capability *after* I added the workaround. Time to check if it's still valid?

Reply:
nope, I've added this explicit, because the lib doen't support this scenario